runtime/oti/j9nonbuilder.h

```diff
@@ -5305,6 +5308,8 @@ typedef struct J9InternalVMFunctions {
 	void (*jfrExecutionSample)(struct J9VMThread *currentThread, struct J9VMThread *sampleThread);
 	jboolean (*setJFRRecordingFileName)(struct J9JavaVM *vm, char *fileName);
 	void (*tearDownJFR)(struct J9JavaVM *vm);
+	jlong (*getTypeIDUTF8)(struct J9VMThread *currentThread, const struct J9UTF8 *className);
+	jlong (*getTypeID)(struct J9VMThread *currentThread, struct J9Class *className);
```

Comment:
The parameter name is misleading; suggest `clazz` instead of `className` (consistent with the definition).